doc/tech/README.md

```diff
@@ -3,203 +3,166 @@
-
-<p align="center"><img src="./figures/tech_vtd.png" alt="virtio" title="Document Types" width="50%"/></p>
-<p align="center"><b>Figure 5:</b> Direct Assignment with Virtual Technology.</p>
-
-<a name="4.1.3"></a>
-#### 4.1.3 Device Sharing with SR-IOV & IOMMU.
-
-This method partitions a hardware device into multiple regions (known as VFs), and uses Direct Assignment to provide workloads exclusive access to one or more of those regions (VFs), thereby bypassing the hypervisor and simultaneously allowing multiple workloads to share the same device.
-
-For this method to be possible, the IO device need to support Single Root Input Output Virtualization (SR-IOV) which allows it to present itself as multiple devices, known as Physical Functions, PFs, and Virtual Functions, VFs as presented in **Figure 6**.
-
-Each of those Virtual Functions can then be independently assigned exclusively to a workload (with the appropriate hardware support of an IOMMU).
+<a name="2.0"></a>
+# 2. Principles
 
-Similar to the previous method ("Direct Assignment"), this method provides better performance than para-virtualization, but lacks the flexibility and the portability sought and therefore must also not be used unless explicitly allowed as an exception as part of the transitional plan adopted by CNTT.
+Any specification work created within CNTT **must** conform to set of principles specified by CNTT:
 
-<p align="center"><img src="./figures/tech_sriov.png" alt="sriov" title="Document Types" width="50%"/></p>
-<p align="center"><b>Figure 6:</b> Device Sharing with SR-IOV & Direct Assignment.</p>
+* [Reference Model Principles](../ref_model/chapters/chapter01.md#13-principles)
+* [Reference Architecture Principles](../ref_arch#principles)
 
-<a name="4.1.4"></a>
-#### 4.1.4 Para-Virtualization method (Hardware support)
+<a name="3.0"></a>
+# 3. Scope
 
-This method basically is a mixture between the software only para-virtualization method and the direct assignment method (including the device sharing method) where the frontEnd driver which is running on the workload is a standard off the shelf driver and the backEnd driver is implemented straight in hardware logic (bypassing the hypervisor with hardware support from an IOMMU and SR-IOV) as shown in **Figure 7**.
+Within the framework of the Common Telecom cloud infrastructure vision, there are three levels of documents needed to describe components and allow the practical application of the systems. They are, as highlighted in **Figure 1**:  **Reference Model**, **Reference Architecture**, **Reference Implementation**, and **Reference Conformance**.
```

Comment:
Is it 3 or 4 ? :)

Reply:
Change to 4, but you also need to add qualification to the list. You could amend it to read:

"...there are _four_ levels of documents needed to describe the components, _realize_ the practical application of the systems _and qualify the resulting cloud infrastructure_."

Reply:
well spotted. fixed in: https://github.com/cntt-n/CNTT/pull/1267/commits/7282c5f5b969ac74f89e812bf9e6cf0784fbb580